ddtrace/contrib/trace_utils.py

```diff
@@ -102,13 +120,28 @@ def _store_headers(headers, span, integration_config, request_or_response):
         return
 
     for header_name, header_value in headers.items():
+        """config._header_tag_name get a element of the dictionary in config.http._header_tags
+        witch get the value from DD_TRACE_HEADER_TAGS environment variable."""
```

Comment:
```suggestion
        which gets the value from DD_TRACE_HEADER_TAGS environment variable."""
```

Reply:
Should this doc string go under the _header_tag_name method declaration? I don't know what the correct practice is, but it seems a bit odd to say what a method does where you use it, but not where it's actually defined.